plugins/org/src/components/Cards/OwnershipCard/ComponentsGrid.tsx

```diff
@@ -44,8 +44,16 @@ const useStyles = makeStyles((theme: BackstageTheme) =>
       },
       height: '100%',
     },
-    bold: {
+    title: {
       fontWeight: theme.typography.fontWeightBold,
+      textAlign: 'center',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
```

Comment:
Honestly it would be pretty cool to try out shrinking the font size to fit, but doing it properly essentially requires reimplementing `react-test-truncate` which the `OverflowTooltip` uses as far as I can tell 😅. Wonder if there's some super simple version that could be used that is just a simple best effort text length check between two different font sizes or something like that, still keeping the truncation.

Reply:
Yes, I also played around with [react-scale-text](https://www.npmjs.com/package/react-scale-text?activeTab=explore), but this didn't work super smoothly/ bring a lot of value in my opinion. I added in a really simple fontSize change to the `ComponentsGrid` but not to the `MembershipListCard` because I thought this didn't look as clean (screenshot below), but could easily add it in to that component as well! 
<img width="636" alt="Screenshot 2023-02-16 at 3 24 28 PM" src="https://user-images.githubusercontent.com/72715376/219492891-89770cbc-245f-4574-8701-bab9573a0615.png">